lite/examples/object_detection/raspberry_pi/detect_usbwebcam.py

```diff
@@ -0,0 +1,158 @@
+# python3
+#
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Example using TF Lite to detect objects with the Raspberry USB webcam."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import io
+import re
+import time
+
+
+import numpy as np
+import cv2
+
+from tflite_runtime.interpreter import Interpreter
+
+CAMERA_WIDTH = 640
+CAMERA_HEIGHT = 480
+
+
+def load_labels(path):
+  """Loads the labels file. Supports files with or without index numbers."""
+  with open(path, 'r', encoding='utf-8') as f:
+    lines = f.readlines()
+    labels = {}
+    for row_number, content in enumerate(lines):
+      pair = re.split(r'[:\s]+', content.strip(), maxsplit=1)
+      if len(pair) == 2 and pair[0].strip().isdigit():
+        labels[int(pair[0])] = pair[1].strip()
+      else:
+        labels[row_number] = pair[0].strip()
+  return labels
+
+
+def set_input_tensor(interpreter, image):
```

Comment:
what about adding an argument for the index, so it's more visible from the calling site?

Reply:
I agree, but I didn't modify the code of the original because I didn't want to repent.